moved/src/state_actor.rs

```diff
@@ -650,11 +650,17 @@ impl<
     > StateActor<S, P, H, R, G, L1G, L2G, B, Q, M, InMemoryStateQueries>
 {
     pub fn on_tx_in_memory() -> OnTx<Self> {
-        Box::new(|| Box::new(|state, changes| state.state_queries.add(changes)))
+        Box::new(|| Box::new(|_state, _changes| ()))
```

Comment:
Is this method not used any more? just returning `()`

Reply:
No longer used, correct. But I am hesitant to remove it in case we end up needing it later. We are still iterating on this after all. Once we have something stable, if this is no longer used then we can remove it entirely.

Reply:
That's valid, we should keep it then ✓